libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -381,16 +381,27 @@ public void initialize(OnNavigationReadyCallback onNavigationReadyCallback) {
 
   /**
    * Gives the ability to manipulate the map directly for anything that might not currently be
-   * supported. This returns null until the view is initialized
+   * supported. This returns null until the view is initialized.
+   * <p>
+   * The {@link NavigationMapboxMap} gives direct access to the map UI (location marker, route, etc.).
    *
-   * @return mapbox map object, or null if view has not been initialized
+   * @return navigation mapbox map object, or null if view has not been initialized
    */
   @Nullable
-  public MapboxMap retrieveMapboxMap() {
```

Comment:
Are we ok here breaking SemVer?
If so, we should document this in the release notes.

Reply:
Yep, will update release notes / documentation